tests/integration/targets/ios_vlans/tasks/main.yaml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Set a fact for 'run_test'
+  ansible.builtin.set_fact:
+    run_test: false
```

Comment:
We should look for a better way to do it, skip at the test invocation level, or maybe look at tags.

Reply:
SKIP whole